languages/python/django-oso/tests/test_django_oso.py

```diff
@@ -144,19 +144,32 @@ def test_partial(rf, partial_policy):
     authorize_filter = authorize_model(request, action="get", model=Post)
     assert (
         str(authorize_filter)
-        == "(AND: ('is_private', False), ('timestamp__gt', 0), ('option', None))"
+        == "(AND: (NOT (AND: ('pk__in', []))), ('is_private', False), ('timestamp__gt', 0), ('option', None))"
     )
 
     q = Post.objects.filter(authorize_filter)
+    assert (
+        str(q.query)
+        == 'SELECT "test_app_post"."id", "test_app_post"."is_private", "test_app_post"."name",'
```

Comment:
Where is the `NOT pk__in` in this SQL?

Reply:
`(AND: ('pk__in', []))` translates to "always false"; `(NOT (AND: ('pk__in', [])))` translates to "always true", so there's no reason for it to appear in the SQL output. The Django ORM has optimized the trivial truth constraint away.